src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -250,4 +250,34 @@ public static Meeting parseMeeting(String meeting) throws ParseException {
         }
         return meetingList;
     }
+
+    /**
+     * Parses a {@code String shortcutName}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code shortcutName} is invalid.
+     */
+    public static String parseShortcutName(String shortcutName) throws ParseException {
+        requireNonNull(shortcutName);
+        String trimmedShortcutName = shortcutName.trim();
+        if (!Shortcut.isValidShortcutName(trimmedShortcutName)) {
+            throw new ParseException(Shortcut.MESSAGE_NAME_CONSTRAINTS);
+        }
+        return trimmedShortcutName;
+    }
+
+    /**
+     * Parses a {@code String shortcutCommand}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code shortcutCommand} is invalid.
+     */
+    public static String parseShortcutCommand(String shortcutCommand) throws ParseException {
+        requireNonNull(shortcutCommand);
+        String trimmedShortcutCommand = shortcutCommand.trim();
+        if (!Shortcut.isValidShortcutCommand(trimmedShortcutCommand)) {
+            throw new ParseException(Shortcut.MESSAGE_COMMAND_CONSTRAINTS);
+        }
+        return trimmedShortcutCommand;
+    }
```

Comment:
For these 2 methods, I thought it might be slightly inconsistent that all the other methods in ParserUtil class named parse... parses some string into an object, while these two methods trims whitespace and checks validity. I personally suggest renaming them to isValidShortcutName/Command. What do you think?